### DISCUSSION TITLE
对于产生的脚手架，我无法将其成环

### DISCUSSION THREAD

**Discussion by 728836672** [Kinggerm/GetOrganelle#357] [Q&A]
您好，金博士，我的多个样本产生了这样的问题，结果没有成环而是出现了两个脚手架，我优化参数并尝试了多次，但好像没有什么变化。非常需要您的帮助以解决这个问题。
[output.zip](https://github.com/user-attachments/files/17709717/output.zip)

**Comment by JianjunJin** [comment]
请根据模板提问。
另外，是否已经参考https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph 中的第二条进行参数调整，根据目前的log文件，并没有全部的数据被使用到。
不完整的问题不一定总能解决，也会跟测序数据有关，不涉及bug和feature的问题，请移至Discussion的Q&A进行提问，谢谢！
